venv/lib/python3.9/site-packages/pip/_internal/commands/debug.py

```
@@ -0,0 +1,215 @@
+
+    usage = """
+      %prog <options>"""
+    ignore_require_venv = True
+
+    def add_options(self):
+        # type: () -> None
+        cmdoptions.add_target_python_options(self.cmd_opts)
+        self.parser.insert_option_group(0, self.cmd_opts)
+        self.parser.config.load()
+
+    def run(self, options, args):
```

Ruff (0.14.8)

Unused method argument: `args`

(ARG002)